training/metabolomics/F1000_Metabolomics_Query_Tabular/F1000_Metabolomics_Query_Tabular_Mass_Adjustment.ga

```diff
@@ -0,0 +1,283 @@
+        {
+          "label": null,
+          "output_name": "output",
+          "uuid": "e6ca3650-5bf0-438d-981c-ecb5c51cbf47"
+        }
+      ]
+    },
+    "1": {
+      "annotation": "",
+      "content_id": null,
+      "errors": null,
+      "id": 1,
+      "input_connections": {},
+      "inputs": [],
+      "label": "sample_metadata",
+      "name": "Input dataset",
+      "outputs": [],
+      "position": {
+        "left": 383,
+        "top": 367
+      },
+      "tool_id": null,
+      "tool_state": "{}",
+      "tool_version": null,
+      "type": "data_input",
+      "uuid": "48565a0d-432b-4211-98f0-19ea37fc4cc7",
+      "workflow_outputs": [
+        {
+          "label": null,
+          "output_name": "output",
+          "uuid": "3056d917-8c84-48ba-9265-916622f1fdb9"
+        }
+      ]
+    },
+    "2": {
+      "annotation": "",
+      "content_id": null,
+      "errors": null,
+      "id": 2,
+      "input_connections": {},
+      "inputs": [],
+      "label": "variable_metadata",
+      "name": "Input dataset",
+      "outputs": [],
+      "position": {
+        "left": 409,
+        "top": 476
+      },
+      "tool_id": null,
+      "tool_state": "{}",
+      "tool_version": null,
+      "type": "data_input",
+      "uuid": "1178cd10-d91a-4382-8006-6d95ad4ac366",
+      "workflow_outputs": [
+        {
+          "label": null,
+          "output_name": "output",
+          "uuid": "aa65b6dc-87e5-44ce-80a0-a47be5078baa"
+        }
+      ]
+    },
+    "3": {
+      "annotation": "",
+      "content_id": "testtoolshed.g2.bx.psu.edu/repos/eslerm/vkmz_qt/vkmz_qt/1.3.1",
+      "errors": null,
+      "id": 3,
+      "input_connections": {
+        "mode|datamatrix": {
+          "id": 0,
+          "output_name": "output"
+        },
+        "mode|samplemetadata": {
+          "id": 1,
+          "output_name": "output"
+        },
+        "mode|variablemetadata": {
+          "id": 2,
+          "output_name": "output"
+        }
+      },
+      "inputs": [
+        {
+          "description": "runtime parameter for tool vkmz_qt",
+          "name": "mode"
+        },
+        {
+          "description": "runtime parameter for tool vkmz_qt",
+          "name": "mode"
+        },
+        {
+          "description": "runtime parameter for tool vkmz_qt",
+          "name": "mode"
+        }
+      ],
+      "label": null,
+      "name": "vkmz_qt",
+      "outputs": [
+        {
+          "name": "output",
+          "type": "tabular"
+        },
+        {
+          "name": "output_html",
+          "type": "html"
+        }
+      ],
+      "position": {
+        "left": 638,
+        "top": 365.5
+      },
+      "post_job_actions": {
+        "HideDatasetActionoutput_html": {
+          "action_arguments": {},
+          "action_type": "HideDatasetAction",
+          "output_name": "output_html"
+        }
+      },
+      "tool_id": "testtoolshed.g2.bx.psu.edu/repos/eslerm/vkmz_qt/vkmz_qt/1.3.1",
```

Comment:
same here .... please make sure this is running against EU and let us know which tools you need.